infra/stack.json

```diff
@@ -0,0 +1,415 @@
+                        "SubnetId": {"Ref": "pubsubnet1"}
+                }
+        },
+        "natgateway2": {
+                "Type": "AWS::EC2::NatGateway",
+                "Properties": {
+                        "AllocationId": {"Fn::GetAtt": ["natgateway2ip", "AllocationId"]},
+                        "SubnetId": {"Ref": "pubsubnet2"}
+                }
+        },
+        "routetable" : {
+           "Type" : "AWS::EC2::RouteTable",
+           "Properties" : {
+              "VpcId" : { "Ref" : "vpc" }
+           }
+        },
+        "internetroute": {
+           "Type" : "AWS::EC2::Route",
+           "DependsOn": ["attachgateway"],
+           "Properties": {
+                   "GatewayId": { "Ref": "ig" },
+                   "RouteTableId": { "Ref": "routetable" },
+                   "DestinationCidrBlock": "0.0.0.0/0"
+           }
+        },
+        "pubsubnet1igassn": {
+           "Type" : "AWS::EC2::SubnetRouteTableAssociation",
+           "Properties" : {
+              "SubnetId" : { "Ref" : "pubsubnet1" },
+              "RouteTableId" : { "Ref" : "routetable" }
+           }
+        },
+        "pubsubnet2igassn": {
+           "Type" : "AWS::EC2::SubnetRouteTableAssociation",
+           "Properties" : {
+              "SubnetId" : { "Ref" : "pubsubnet2" },
+              "RouteTableId" : { "Ref" : "routetable" }
+           }
+        },
+        "privroutetable1" : {
+           "Type" : "AWS::EC2::RouteTable",
+           "Properties" : {
+              "VpcId" : { "Ref" : "vpc" }
+           }
+        },
+        "privroutetable2" : {
+           "Type" : "AWS::EC2::RouteTable",
+           "Properties" : {
+              "VpcId" : { "Ref" : "vpc" }
+           }
+        },
+        "natroute1": {
+           "Type" : "AWS::EC2::Route",
+           "DependsOn": ["attachgateway"],
+           "Properties": {
+                   "RouteTableId": { "Ref": "privroutetable1" },
+                   "DestinationCidrBlock": "0.0.0.0/0",
+                   "NatGatewayId": { "Ref": "natgateway1" }
+           }
+        },
+        "natroute2": {
+           "Type" : "AWS::EC2::Route",
+           "DependsOn": ["attachgateway"],
+           "Properties": {
+                   "RouteTableId": { "Ref": "privroutetable2" },
+                   "DestinationCidrBlock": "0.0.0.0/0",
+                   "NatGatewayId": { "Ref": "natgateway2" }
+           }
+        },
+        "privsubnet1rtassn": {
+           "Type" : "AWS::EC2::SubnetRouteTableAssociation",
+           "Properties" : {
+              "SubnetId" : { "Ref" : "privsubnet1" },
+              "RouteTableId" : { "Ref" : "privroutetable1" }
+           }
+        },
+        "privsubnet2rtassn": {
+           "Type" : "AWS::EC2::SubnetRouteTableAssociation",
+           "Properties" : {
+              "SubnetId" : { "Ref" : "privsubnet2" },
+              "RouteTableId" : { "Ref" : "privroutetable2" }
+           }
+        },
+        "subnetgroup" : {
```

Comment:
**Naming Suggestion** (non-blocking): This is specific to db, so maybe call it `dbsubnetgroup`?

Reply:
Done.